pkg/instrumentation/metrics.go

```diff
@@ -1,32 +1,37 @@
 package instrumentation
 
 import (
+	"time"
+
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/collectors"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
 // TODO Update metric names according to: https://prometheus.io/docs/instrumenting/writing_exporters/#naming
 const (
-	NameSpace                                            = "content_sources"
-	HttpStatusHistogram                                  = "http_status_histogram"
-	RepositoriesTotal                                    = "repositories_total"
-	RepositoryConfigsTotal                               = "repository_configs_total"
-	PublicRepositoriesNotIntrospectedLast24HoursTotal    = "public_repositories_not_introspected_last_24_hours_total"
-	PublicRepositoriesWithFailedIntrospectionTotal       = "public_repositories_with_failed_introspection_total"
-	NonPublicRepositoriesNotIntrospectedLast24HoursTotal = "non_public_repositories_not_introspected_last_24_hours_total"
-	Top50Repositories                                    = "top_50_repositories"
+	NameSpace                                      = "content_sources"
+	HttpStatusHistogram                            = "http_status_histogram"
+	RepositoriesTotal                              = "repositories_total"
+	RepositoryConfigsTotal                         = "repository_configs_total"
+	PublicRepositories36HourIntrospectionTotal     = "public_repositories_36_hour_introspection_total"
+	PublicRepositoriesWithFailedIntrospectionTotal = "public_repositories_with_failed_introspection_total"
+	CustomRepositories36HourIntrospectionTotal     = "custom_repositories_36_hour_introspection_total"
+	KafkaMessageLatency                            = "kafka_message_latency"
+	KafkaMessageStatus                             = "kafka_message_status"
```

Comment:
`KafkaMessageResultTotal` might be a better name, since this is a counter of the result.